packages/matter.js/src/behavior/internal/Reactors.ts

```diff
@@ -211,6 +211,10 @@ class ReactorBacking<T extends any[], R> {
         if (!(error instanceof Error)) {
             error = new Error(`${error}`);
         }
+        if (error instanceof MatterError) {
+            logger.error("Unhandled MatterError", this.#augmentError(error));
+            throw error;
```

Comment:
If the context is "offline" there may be nobody listening to catch the error -- so this would be unhandled and crash the process.  So maybe rethrow could have additional conditions added?

Reply:
This I tried but then listeners registered eg in initialize of a cluster that always are offline are seen as offline rather then "executed online as change handler but the reactor specified offline so the online info is lost here - or I used the wrong flag. This#offline was not working as expected in this case.

Reply:
Ok with  new approach reverted

Reply:
Reverted